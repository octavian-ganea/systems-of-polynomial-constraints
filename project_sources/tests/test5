x^3 + y^3 - 1 = 0
x^2 + y^2 - 1 = 0
x + y > 0
2*x + y - 2 < 0