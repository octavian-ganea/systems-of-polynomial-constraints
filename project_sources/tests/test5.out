Parsing file test5
Parsing succeded. Results:
List of all variables:
Vector(x, y)
List of all polynomials:
Vector((1.0 y^3 + -1.0 ) + 1.0 x^3, (1.0 y^2 + -1.0 ) + 1.0 x^2, (1.0 y) + 1.0 x, (1.0 y + -2.0 ) + 2.0 x)

List of desired signs : Vector(=, =, >, <)
Analysed polynomials: 
(1.0 y^3 + -1.0 ) + 1.0 x^3
(1.0 y^2 + -1.0 ) + 1.0 x^2
(1.0 y) + 1.0 x
(1.0 y + -2.0 ) + 2.0 x
vars: Vector(x, y)

Univariate projection (on real axis) contains these polynomials: 
1.0 y^3 + -1.0 
1.0 y^2 + -1.0 
1.0 y
1.0 y + -2.0 
27.0  + 27.0 y^6 + -54.0 y^3
4.0 y^2 + -4.0 
3.0 y^2 + 2.0 y^6 + -3.0 y^4 + -2.0 y^3
-1.0 y^2 + 1.0 
1.0  + 1.0 y^6 + -2.0 y^3
-1.0  + 3.0 y^2 + 1.0 y^6 + -3.0 y^4
1.0 y^3
-1.0  + 2.0 y^2
1.0 y^2
-12.0 y + 7.0 y^3 + 6.0 y^2
-8.0  + 12.0 y + 1.0 y^3 + -6.0 y^2
5.0 y^2 + -4.0 y
4.0  + 1.0 y^2 + -4.0 y
1.0 y + 2.0 

There are 289 cells :
Map(x -> 1.5, y -> -3.0)
Map(x -> 2.5, y -> -3.0)
Map(x -> 2.75, y -> -3.0)
Map(x -> 3.0, y -> -3.0)
Map(x -> 3.0182944859378313, y -> -3.0)
Map(x -> 3.0365889718756627, y -> -3.0)
Map(x -> 4.036588971875663, y -> -3.0)
Map(x -> 1.0, y -> -2.0)
Map(x -> 2.0, y -> -2.0)
Map(x -> 2.040041911525952, y -> -2.0)
Map(x -> 2.080083823051904, y -> -2.0)
Map(x -> 3.080083823051904, y -> -2.0)
Map(x -> 0.9031179117563641, y -> -1.9031179117563641)
Map(x -> 1.9031179117563641, y -> -1.9031179117563641)
Map(x -> 1.927338433817273, y -> -1.9031179117563641)
Map(x -> 1.951558955878182, y -> -1.9031179117563641)
Map(x -> 1.9712936531733676, y -> -1.9031179117563641)
Map(x -> 1.9910283504685533, y -> -1.9031179117563641)
Map(x -> 2.9910283504685533, y -> -1.9031179117563641)
Map(x -> 0.806235823512728, y -> -1.806235823512728)
Map(x -> 1.806235823512728, y -> -1.806235823512728)
Map(x -> 1.854676867634546, y -> -1.806235823512728)
Map(x -> 1.9031179117563641, y -> -1.806235823512728)
Map(x -> 1.9031179118593107, y -> -1.806235823512728)
Map(x -> 1.9031179119622572, y -> -1.806235823512728)
Map(x -> 2.903117911962257, y -> -1.806235823512728)
Map(x -> 0.4031429084485354, y -> -1.4031429084485354)
Map(x -> 1.4031429084485354, y -> -1.4031429084485354)
Map(x -> 1.479243246909301, y -> -1.4031429084485354)
Map(x -> 1.5553435853700668, y -> -1.4031429084485354)
Map(x -> 1.6284575197971671, y -> -1.4031429084485354)
Map(x -> 1.7015714542242677, y -> -1.4031429084485354)
Map(x -> 2.7015714542242675, y -> -1.4031429084485354)
Map(x -> 4.999338434297229E-5, y -> -1.000049993384343)
Map(x -> 1.000049993384343, y -> -1.000049993384343)
Map(x -> 1.1300012689625416, y -> -1.000049993384343)
Map(x -> 1.2599525445407402, y -> -1.000049993384343)
Map(x -> 1.379988770616456, y -> -1.000049993384343)
Map(x -> 1.5000249966921715, y -> -1.000049993384343)
Map(x -> 2.5000249966921713, y -> -1.000049993384343)
Map(x -> 2.4996692171486146E-5, y -> -1.0000249966921715)
Map(x -> 1.0000249966921715, y -> -1.0000249966921715)
Map(x -> 1.1299808968565874, y -> -1.0000249966921715)
Map(x -> 1.2599367970210036, y -> -1.0000249966921715)
Map(x -> 1.3799746476835446, y -> -1.0000249966921715)
Map(x -> 1.5000124983460856, y -> -1.0000249966921715)
Map(x -> 2.5000124983460856, y -> -1.0000249966921715)
Map(x -> -1.0000010008260192, y -> -1.0)
Map(x -> -1.0008260191095398E-6, y -> -1.0)
Map(x -> 0.49999949958699047, y -> -1.0)
Map(x -> 1.0, y -> -1.0)
Map(x -> 1.1299605249474367, y -> -1.0)
Map(x -> 1.2599210498948732, y -> -1.0)
Map(x -> 1.3799605249474367, y -> -1.0)
Map(x -> 1.5, y -> -1.0)
Map(x -> 2.5, y -> -1.0)
Map(x -> -1.5210053831851758, y -> -0.8535533908737843)
Map(x -> -0.5210053831851758, y -> -0.8535533908737843)
Map(x -> -1.823719508742272E-10, y -> -0.8535533908737843)
Map(x -> 0.5210053828204319, y -> -0.8535533908737843)
Map(x -> 0.6872793868471081, y -> -0.8535533908737843)
Map(x -> 0.8535533908737843, y -> -0.8535533908737843)
Map(x -> 1.014231403565321, y -> -0.8535533908737843)
Map(x -> 1.1749094162568576, y -> -0.8535533908737843)
Map(x -> 1.3008430558468749, y -> -0.8535533908737843)
Map(x -> 1.4267766954368921, y -> -0.8535533908737843)
Map(x -> 2.4267766954368923, y -> -0.8535533908737843)
Map(x -> -1.7071067810689762, y -> -0.7071067817475687)
Map(x -> -0.7071067810689762, y -> -0.7071067817475687)
Map(x -> -2.217249162050905E-10, y -> -0.7071067817475687)
Map(x -> 0.7071067806255263, y -> -0.7071067817475687)
Map(x -> 0.7071067811865475, y -> -0.7071067817475687)
Map(x -> 0.7071067817475687, y -> -0.7071067817475687)
Map(x -> 0.9066425355631618, y -> -0.7071067817475687)
Map(x -> 1.1061782893787548, y -> -0.7071067817475687)
Map(x -> 1.2298658401262696, y -> -0.7071067817475687)
Map(x -> 1.3535533908737842, y -> -0.7071067817475687)
Map(x -> 2.3535533908737842, y -> -0.7071067817475687)
Map(x -> -1.935395446852446, y -> -0.3536033913103694)
Map(x -> -0.9353954468524459, y -> -0.3536033913103694)
Map(x -> -0.2908960277710383, y -> -0.3536033913103694)
Map(x -> 0.3536033913103694, y -> -0.3536033913103694)
Map(x -> 0.6444994189909429, y -> -0.3536033913103694)
Map(x -> 0.9353954466715164, y -> -0.3536033913103694)
Map(x -> 0.9749605367398767, y -> -0.3536033913103694)
Map(x -> 1.014525626808237, y -> -0.3536033913103694)
Map(x -> 1.0956636612317108, y -> -0.3536033913103694)
Map(x -> 1.1768016956551848, y -> -0.3536033913103694)
Map(x -> 2.1768016956551848, y -> -0.3536033913103694)
Map(x -> -2.0, y -> -1.0000087317009202E-4)
Map(x -> -1.0, y -> -1.0000087317009202E-4)
Map(x -> -0.49994999956341496, y -> -1.0000087317009202E-4)
Map(x -> 1.0000087317009202E-4, y -> -1.0000087317009202E-4)
Map(x -> 0.5000500004365851, y -> -1.0000087317009202E-4)
Map(x -> 1.0, y -> -1.0000087317009202E-4)
Map(x -> 1.0000250002182924, y -> -1.0000087317009202E-4)
Map(x -> 1.000050000436585, y -> -1.0000087317009202E-4)
Map(x -> 2.000050000436585, y -> -1.0000087317009202E-4)
Map(x -> -2.0, y -> -5.050084959460078E-5)
Map(x -> -1.0, y -> -5.050084959460078E-5)
Map(x -> -0.4999747495752027, y -> -5.050084959460078E-5)
Map(x -> 5.050084959460078E-5, y -> -5.050084959460078E-5)
Map(x -> 0.5000252504247973, y -> -5.050084959460078E-5)
Map(x -> 1.0, y -> -5.050084959460078E-5)
Map(x -> 1.0000126252123986, y -> -5.050084959460078E-5)
Map(x -> 1.0000252504247973, y -> -5.050084959460078E-5)
Map(x -> 2.0000252504247973, y -> -5.050084959460078E-5)
Map(x -> -2.0, y -> -1.0008260191095398E-6)
Map(x -> -1.0, y -> -1.0008260191095398E-6)
Map(x -> -0.49999949958699047, y -> -1.0008260191095398E-6)
Map(x -> 1.0008260191095398E-6, y -> -1.0008260191095398E-6)
Map(x -> 0.5000005004130096, y -> -1.0008260191095398E-6)
Map(x -> 1.0, y -> -1.0008260191095398E-6)
Map(x -> 1.0000002502065048, y -> -1.0008260191095398E-6)
Map(x -> 1.0000005004130095, y -> -1.0008260191095398E-6)
Map(x -> 2.0000005004130097, y -> -1.0008260191095398E-6)
Map(x -> -2.0, y -> -7.890987864905056E-7)
Map(x -> -1.0, y -> -7.890987864905056E-7)
Map(x -> -0.49999960545060673, y -> -7.890987864905056E-7)
Map(x -> 7.890987864905056E-7, y -> -7.890987864905056E-7)
Map(x -> 0.5000003945493933, y -> -7.890987864905056E-7)
Map(x -> 1.0, y -> -7.890987864905056E-7)
Map(x -> 1.0000001972746966, y -> -7.890987864905056E-7)
Map(x -> 1.0000003945493932, y -> -7.890987864905056E-7)
Map(x -> 2.000000394549393, y -> -7.890987864905056E-7)
Map(x -> -2.0, y -> -5.773715538714714E-7)
Map(x -> -1.0, y -> -5.773715538714714E-7)
Map(x -> -0.49999971131422305, y -> -5.773715538714714E-7)
Map(x -> 5.773715538714714E-7, y -> -5.773715538714714E-7)
Map(x -> 0.500000288685777, y -> -5.773715538714714E-7)
Map(x -> 1.0, y -> -5.773715538714714E-7)
Map(x -> 1.0000001443428883, y -> -5.773715538714714E-7)
Map(x -> 1.0000002886857768, y -> -5.773715538714714E-7)
Map(x -> 2.0000002886857766, y -> -5.773715538714714E-7)
Map(x -> -2.0, y -> -2.886857769357357E-7)
Map(x -> -1.0, y -> -2.886857769357357E-7)
Map(x -> -0.4999998556571115, y -> -2.886857769357357E-7)
Map(x -> 2.886857769357357E-7, y -> -2.886857769357357E-7)
Map(x -> 0.5000001443428884, y -> -2.886857769357357E-7)
Map(x -> 1.0, y -> -2.886857769357357E-7)
Map(x -> 1.0000000721714444, y -> -2.886857769357357E-7)
Map(x -> 1.0000001443428885, y -> -2.886857769357357E-7)
Map(x -> 2.0000001443428888, y -> -2.886857769357357E-7)
Map(x -> -2.0, y -> 0.0)
Map(x -> -1.0, y -> 0.0)
Map(x -> -0.5, y -> 0.0)
Map(x -> 0.0, y -> 0.0)
Map(x -> 0.5, y -> 0.0)
Map(x -> 1.0, y -> 0.0)
Map(x -> 2.0, y -> 0.0)
Map(x -> -1.935414347029778, y -> 0.35355339059327373)
Map(x -> -0.935414347029778, y -> 0.35355339059327373)
Map(x -> -0.6444838688559087, y -> 0.35355339059327373)
Map(x -> -0.3535533906820395, y -> 0.35355339059327373)
Map(x -> 0.23483495701066182, y -> 0.35355339059327373)
Map(x -> 0.8232233047033631, y -> 0.35355339059327373)
Map(x -> 0.8793188256984242, y -> 0.35355339059327373)
Map(x -> 0.9354143466934853, y -> 0.35355339059327373)
Map(x -> 0.9602302255416031, y -> 0.35355339059327373)
Map(x -> 0.985046104389721, y -> 0.35355339059327373)
Map(x -> 1.985046104389721, y -> 0.35355339059327373)
Map(x -> -1.7071067817475687, y -> 0.7071067811865475)
Map(x -> -0.7071067817475687, y -> 0.7071067811865475)
Map(x -> -0.7071067814759482, y -> 0.7071067811865475)
Map(x -> -0.7071067812043278, y -> 0.7071067811865475)
Map(x -> -0.030330085898800763, y -> 0.7071067811865475)
Map(x -> 0.6464466094067263, y -> 0.7071067811865475)
Map(x -> 0.676776695296637, y -> 0.7071067811865475)
Map(x -> 0.7071067811865476, y -> 0.7071067811865475)
Map(x -> 0.7858822478706919, y -> 0.7071067811865475)
Map(x -> 0.8646577145548363, y -> 0.7071067811865475)
Map(x -> 1.8646577145548364, y -> 0.7071067811865475)
Map(x -> -1.7535533910065233, y -> 0.7535533905932736)
Map(x -> -0.7535533910065234, y -> 0.7535533905932736)
Map(x -> -0.7054700510497485, y -> 0.7535533905932736)
Map(x -> -0.6573867110929736, y -> 0.7535533905932736)
Map(x -> -0.01708170319480523, y -> 0.7535533905932736)
Map(x -> 0.6232233047033632, y -> 0.7535533905932736)
Map(x -> 0.6403050077474749, y -> 0.7535533905932736)
Map(x -> 0.6573867107915866, y -> 0.7535533905932736)
Map(x -> 0.7437691142198057, y -> 0.7535533905932736)
Map(x -> 0.8301515176480249, y -> 0.7535533905932736)
Map(x -> 1.8301515176480248, y -> 0.7535533905932736)
Map(x -> -1.8000000000611696, y -> 0.7999999999999999)
Map(x -> -0.8000000000611696, y -> 0.7999999999999999)
Map(x -> -0.7000000001608804, y -> 0.7999999999999999)
Map(x -> -0.6000000002605912, y -> 0.7999999999999999)
Map(x -> -1.3029555212540345E-10, y -> 0.7999999999999999)
Map(x -> 0.6000000000000001, y -> 0.7999999999999999)
Map(x -> 0.6936497183102174, y -> 0.7999999999999999)
Map(x -> 0.7872994366204347, y -> 0.7999999999999999)
Map(x -> 1.7872994366204347, y -> 0.7999999999999999)
Map(x -> -1.8745464835043946, y -> 0.8745464829280681)
Map(x -> -0.8745464835043946, y -> 0.8745464829280681)
Map(x -> -0.6797440901255156, y -> 0.8745464829280681)
Map(x -> -0.4849416967466367, y -> 0.8745464829280681)
Map(x -> -2.338457205652844E-11, y -> 0.8745464829280681)
Map(x -> 0.48494169669986753, y -> 0.8745464829280681)
Map(x -> 0.5238342276179166, y -> 0.8745464829280681)
Map(x -> 0.5627267585359659, y -> 0.8745464829280681)
Map(x -> 0.6272747333496355, y -> 0.8745464829280681)
Map(x -> 0.691822708163305, y -> 0.8745464829280681)
Map(x -> 1.691822708163305, y -> 0.8745464829280681)
Map(x -> -1.9490929661141225, y -> 0.9490929658561363)
Map(x -> -0.9490929661141225, y -> 0.9490929658561363)
Map(x -> -0.6320445325663255, y -> 0.9490929658561363)
Map(x -> -0.31499609901852854, y -> 0.9490929658561363)
Map(x -> -1.8610377128247535E-10, y -> 0.9490929658561363)
Map(x -> 0.314996098646321, y -> 0.9490929658561363)
Map(x -> 0.4202248078591265, y -> 0.9490929658561363)
Map(x -> 0.5254535170719321, y -> 0.9490929658561363)
Map(x -> 1.525453517071932, y -> 0.9490929658561363)
Map(x -> -1.9745464701721231, y -> 0.9745464695165569)
Map(x -> -0.974546470172123, y -> 0.9745464695165569)
Map(x -> -0.5993660304733157, y -> 0.9745464695165569)
Map(x -> -0.22418559077450845, y -> 0.9745464695165569)
Map(x -> -3.9933503725997355E-10, y -> 0.9745464695165569)
Map(x -> 0.22418558997583837, y -> 0.9745464695165569)
Map(x -> 0.32241866443128553, y -> 0.9745464695165569)
Map(x -> 0.4206517388867327, y -> 0.9745464695165569)
Map(x -> 0.4666892520642271, y -> 0.9745464695165569)
Map(x -> 0.5127267652417216, y -> 0.9745464695165569)
Map(x -> 1.5127267652417216, y -> 0.9745464695165569)
Map(x -> -1.9999999737134122, y -> 0.9999999731769774)
Map(x -> -0.9999999737134122, y -> 0.9999999731769774)
Map(x -> -0.5001157961504267, y -> 0.9999999731769774)
Map(x -> -2.3161858744123677E-4, y -> 0.9999999731769774)
Map(x -> -1.2148837593074129E-9, y -> 0.9999999731769774)
Map(x -> 2.3161615767371816E-4, y -> 0.9999999731769774)
Map(x -> 0.002274445279616291, y -> 0.9999999731769774)
Map(x -> 0.004317274401558864, y -> 0.9999999731769774)
Map(x -> 0.25215864390653503, y -> 0.9999999731769774)
Map(x -> 0.5000000134115112, y -> 0.9999999731769774)
Map(x -> 1.5000000134115112, y -> 0.9999999731769774)
Map(x -> -1.9999999868678833, y -> 0.9999999865884888)
Map(x -> -0.9999999868678832, y -> 0.9999999865884888)
Map(x -> -0.5000818840887076, y -> 0.9999999865884888)
Map(x -> -1.6378130953188238E-4, y -> 0.9999999865884888)
Map(x -> -1.976904211837237E-9, y -> 0.9999999865884888)
Map(x -> 1.637773557234587E-4, y -> 0.9999999865884888)
Map(x -> 0.001795200164256884, y -> 0.9999999865884888)
Map(x -> 0.003426622972790309, y -> 0.9999999865884888)
Map(x -> 0.25171331483927295, y -> 0.9999999865884888)
Map(x -> 0.5000000067057556, y -> 0.9999999865884888)
Map(x -> 1.5000000067057555, y -> 0.9999999865884888)
Map(x -> -2.0, y -> 1.0)
Map(x -> -1.0, y -> 1.0)
Map(x -> -0.5000500004365851, y -> 1.0)
Map(x -> -1.0000087317009202E-4, y -> 1.0)
Map(x -> -5.050084959460078E-5, y -> 1.0)
Map(x -> -1.0008260191095398E-6, y -> 1.0)
Map(x -> 0.24999949958699044, y -> 1.0)
Map(x -> 0.5, y -> 1.0)
Map(x -> 1.5, y -> 1.0)
Map(x -> -2.4999950826208526, y -> 1.4999950825820623)
Map(x -> -1.4999950826208526, y -> 1.4999950825820623)
Map(x -> -1.4170948459648405, y -> 1.4999950825820623)
Map(x -> -1.3341946093088282, y -> 1.4999950825820623)
Map(x -> -0.5420960752999296, y -> 1.4999950825820623)
Map(x -> 0.25000245870896887, y -> 1.4999950825820623)
Map(x -> 1.2500024587089689, y -> 1.4999950825820623)
Map(x -> -2.9999901656689385, y -> 1.9999901651641248)
Map(x -> -1.9999901656689387, y -> 1.9999901651641248)
Map(x -> -1.9564552990760058, y -> 1.9999901651641248)
Map(x -> -1.912920432483073, y -> 1.9999901651641248)
Map(x -> -0.9564577575325677, y -> 1.9999901651641248)
Map(x -> 4.917417937622126E-6, y -> 1.9999901651641248)
Map(x -> 1.0000049174179377, y -> 1.9999901651641248)
Map(x -> -2.9999950833423235, y -> 1.9999950825820623)
Map(x -> -1.9999950833423235, y -> 1.9999950825820623)
Map(x -> -1.9564604455406245, y -> 1.9999950825820623)
Map(x -> -1.9129258077389255, y -> 1.9999950825820623)
Map(x -> -0.9564616745149783, y -> 1.9999950825820623)
Map(x -> 2.458708968866574E-6, y -> 1.9999950825820623)
Map(x -> 1.0000024587089689, y -> 1.9999950825820623)
Map(x -> -3.0, y -> 2.0)
Map(x -> -2.0, y -> 2.0)
Map(x -> -1.9564655914905074, y -> 2.0)
Map(x -> -1.9129311829810147, y -> 2.0)
Map(x -> -0.9564655914905074, y -> 2.0)
Map(x -> 0.0, y -> 2.0)
Map(x -> 1.0, y -> 2.0)
Map(x -> -4.0, y -> 3.0)
Map(x -> -3.0, y -> 3.0)
Map(x -> -2.9812480342187, y -> 3.0)
Map(x -> -2.9624960684374004, y -> 3.0)
Map(x -> -1.7312480342751766, y -> 3.0)
Map(x -> -0.5000000001129526, y -> 3.0)
Map(x -> 0.49999999988704735, y -> 3.0)
====== FINAL RESULT FOR test5 ================================
Map(x -> 1.0, y -> -7.890987864905056E-7)
====================================================

